tests/conftest.py

```diff
@@ -333,6 +333,7 @@ def creds(duthost):
     groups.append("fanout")
     logger.info("dut {} belongs to groups {}".format(duthost.hostname, groups))
     files = glob.glob("../ansible/group_vars/all/*.yml")
+    files += glob.glob("../ansible/vars/*.yml")
     for group in groups:
```

Comment:
Is 'all' part of groups list?

Reply:
It is, but this file isn't under 'all'.

Reply:
Also this is `vars` not `group_vars`, that caught me off guard too. :)